### ISSUE TITLE
accessibility with screen reader software

### ISSUE THREAD

**Issue by RealAmethyst** [xemu-project/xemu#2034]
### Bug Description

I am fully blind and use a screen reader to navigate my computer, this software is incompattible with xemu. It does not speak at all when loading the application

### Expected Behavior

I expect  my screen reader to speak out relevant information and speak out information as I navigate the UI

### xemu Version

0.8.39

### System Information

windows 10 22H2

### Additional Context

to test this you can use the default screen reader on windows by holding down ctrl +windows key and pressing enter.

**Comment by mborgerson** [comment]
Hello! I agree xemu should be more accessible, including working with screen readers.

The xemu user interface is developed with the Dear ImGui library, which to my knowledge unfortunately does not work with screen readers yet. I did search to see if they were tracking this feature and found [this thread](https://github.com/ocornut/imgui/issues/4122). This may improve in the future.

As for games themselves running within xemu, it will be challenging to make games truly accessible, as xemu renders images and geometry, not text that can be easily scraped and fed to a text to speech system. However, I believe AI based image recognition and description technology could be really helpful here.

**Comment by RealAmethyst** [comment]
@mborgerson  Games themselves are out of our controll pretty much. Image based description and AI will help a lot, but for me at least, I play fighting games so I just learn the different menus and sounds. It's just the emulator UI itself that is a challenge. Hopefully that gets improved in the future with the UI framework. Alternitivly there's this: https://github.com/nvaccess/nvda/blob/master/extras/controllerClient/readme.md which is something you could implement for teh application. Not sure how challenging that is, but it is an option.
